tool/generate-module-metadata.ts

```diff
@@ -0,0 +1,119 @@
+// Gets list of all functions and variables for the built-in modules
+//
+// Outputs `sass-site/source/assets/js/playground/module-metadata.ts` file, which is omitted from source code and
+// must be built before the `playground` bundle is built with rollup.
+import {compileString, sassTrue} from 'sass';
+import {writeFileSync} from 'fs';
+import path from 'path';
+
+const moduleDescriptions = {
+  color:
+    'generates new colors based on existing ones, making it easy to build color themes',
+  list: 'lets you access and modify values in lists',
+  map: 'makes it possible to look up the value associated with a key in a map, and much more',
+  math: 'provides functions that operate on numbers',
+  meta: 'exposes the details of Sass’s inner workings',
+  selector: 'provides access to Sass’s powerful selector engine',
+  string: 'makes it easy to combine, search, or split apart strings',
+};
+
+const modules = [
+  'color',
+  'list',
+  'map',
+  'math',
+  'meta',
+  'selector',
+  'string',
+] as const;
+
+type ModuleName = (typeof modules)[number];
+
+interface ModuleDefinition {
+  name: ModuleName;
+  description: string;
+  functions?: string[];
+  variables?: string[];
```

Comment:
Modules without functions or variables should have empty lists, not undefined lists.

```suggestion
  functions: string[];
  variables: string[];
```

Reply:
Fixed in https://github.com/sass/sass-site/pull/1166/commits/7c4ef88299d155c3d4db97b36d502220278162a6